libdnf5-cli/output/repo_info.cpp

```diff
@@ -252,7 +252,7 @@ void print_repoinfo_json([[maybe_unused]] const std::vector<std::unique_ptr<IRep
         json_object_object_add(json_repo, "gpg_key", json_gpg_keys);
 
         json_object_object_add(json_repo, "repo_gpgcheck", json_object_new_boolean(repo->get_repo_gpgcheck()));
-        json_object_object_add(json_repo, "gpgcheck", json_object_new_boolean(repo->get_gpgcheck()));
+        json_object_object_add(json_repo, "pkg_gpgcheck", json_object_new_boolean(repo->get_pkg_gpgcheck()));
```

Comment:
This change is breaking CI tests which can be fixed easily [enough](https://github.com/rpm-software-management/ci-dnf-stack/pull/1565) but we should be careful about changing the `json` output.

Reply:
I agree, still I see this machine-readable output functionality new enough to do such changes.